test/e2e/framework/util.go

```diff
@@ -804,29 +703,6 @@ func LogFailedContainers(c clientset.Interface, ns string, logFunc func(ftm stri
 	}
 }
 
-func LogPodsWithLabels(c clientset.Interface, ns string, match map[string]string, logFunc func(ftm string, args ...interface{})) {
```

Comment:
WaitForPodsSuccess is only caller as https://cs.k8s.io/?q=LogPodsWithLabels&i=nope&files=&repos=kubernetes/kubernetes and that is removed as the above.